_posts/2018-05-24-anna-maria-sichani.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Anna-Maria Sichani joins Project Team
-author: Antonio Rojas Castro
+authors: Antonio Rojas Castro
```

Comment:
This should be in this format, even if you only have one author for the post.
authors:
- Maria José Afanador-Llach
- Adam Crymble